resolwe_bio/tools/make_igv_session.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name,import-error
+"""Create IGV session."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import argparse
+from lxml import etree
+
+parser = argparse.ArgumentParser(description='Create igv session')
+
+parser.add_argument('-bam', '--bam_path', nargs='+', default=[], help='list of paths to bam file')
+parser.add_argument('-genomeid', '--genomeid', help='genome')
+parser.add_argument('-locus', '--locus', help='Locus where to open IGV')
```

Comment:
Again, try to come up with a short flag that is not the same as the full argument name.